lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+
+# Human sub-class of Player
+class Players::Human < Player
+
+	# Accepts input from the player for their turn of the game
+	def move(input)
+		input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```